wp-content/civi-extensions/goonjcustom/Civi/DroppingCenterService.php

```diff
@@ -460,6 +460,12 @@ public static function droppingCenterTabset($tabsetName, &$tabs, $context) {
       return;
     }
     $tabConfigs = [
+      'eventCoordinators' => [
+        'title' => ts('event coordinators'),
```

Comment:
```suggestion
        'title' => ts('event coordinators'),
```

Can we use Capital letters here.

Reply:
done